src/app/products/page.js

```diff
@@ -0,0 +1,12 @@
+import Products from '@/components/Products/Products';
+import React from 'react';
+
+const page = () => {
+    return (
+        <div>
+            <Products/>
+        </div>
+    );
```

Comment:
If don't the div then remove it and rename the page with the productPage and use `
const ProductPage = () => ( <Products/>)
export ProductPage 
`